launcher/translations/POTranslator.h

```diff
@@ -9,6 +9,7 @@ class POTranslator : public QTranslator
     Q_OBJECT
 public:
     explicit POTranslator(const QString& filename, QObject * parent = nullptr);
+    virtual ~POTranslator();
```

Comment:
Couldn't d just be changed to a smart pointer instead of manually defining a destructor